pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -244,6 +199,7 @@ def unshare_mounts():
         if ret != 0:
             errno = get_errno_loc()[0]
             raise OSError(errno, os.strerror(errno))
+    subprocess.check_call(["mount", "--make-rslave", "/"])
```

Comment:
Does private → rslave make a difference? I.e. what would mount something inside of that mount? (This should be explained in the commit message)

Reply:
This is so that the long running monitor process can see when the filesystem is mounted "for real" in the parent namespace. Then it will drop its own mount.

Reply:
Thanks!